README.md

```diff
@@ -41,60 +50,40 @@ todoapp/
 └── Makefile
 ```
 
-### Create a new app module
-
-Generates the handler, service, and repository files for a specific app module.
-
-- Create the app handler, service, and repository files inside the `internal/<app_name>/` folder.
-- If you also want to generate data storage implementation (MySQL, PostgreSQL, etc.) for the repository, you can pass a database flag and choose your database.
-
-```bash
-bccli app todo
-```
-
-This will generate
-```bash
-internal/
-├── domain/
-├── infra/
-└── todo/
-    ├── interface/
-    │   └── rest/
-    │       └── todo.go
-    ├── repository/
-    └── service/
-        └── todo.go
-```
 ### Create a domain entity
-
-Creates a domain entity file containing the struct for a given domain name.
-- Creates a new file in the internal/domain/ folder for the given domain.
+The ``domain`` command used to create a new domain entity. This command scaffolds domain files in a structured manner, ensuring separation of concerns and clear layers.
 
 ```bash
-bccli domain todo
+bccli domain generate todo
 ```
-This will generate:
+This command generates a domain for ``todo`` and creates related files in the following structure:
 ```bash
 internal/
 ├── domain/
 │   └── todo.go
+├── todo/
+│   ├── interface/
+│   │   └── rest/
+│   │       └── todo.go
+│   ├── repository/
+│   └── service/
+│       └── todo.go
 └── infra/
 ```
 
 ### Set up Infrastucture
-Sets up the infrastructure for your project.
-- --db: Initializes a database setup file inside the internal/infra/ folder.
+Setting up databases and other infrastructure components with the ``infra`` command. Use it to scaffold infrastructure setup, such as databases, caching systems, or external services:
 
 ```bash
-bccli infra --db=mysql
+bccli infra generate --database=mysql
```

Comment:
its an argument to generate an infra config, not a flag

Reply:
oaky, i just fixed it